src/main/java/com/softserveinc/ita/rozetka/page_objects/BasketPage.java

```diff
@@ -17,32 +16,40 @@ public OrderPage openOrderPage() {
         return new OrderPage();
     }
 
-    public BasketPage increaseProductCount(int numberOfProduct) {
-        $x(format("((//*[@class='button button_color_white" +
-                " button_size_medium cart-counter__button'])[2])[%d]", numberOfProduct)).click();
+    public BasketPage increaseProductCount(int productNumber) {
+        var productSum = $x(format("(//*[@class='cart-product__price'])[%d]", productNumber)).text();
+        $x(format("((//*[@class='button button_color_white button_size_medium cart-counter__button'])[2])[%d]",
+                productNumber)).click();
+        /* checking var "productSum" in the next statement have been added to ensure updating information in the basket window */
```

Comment:
invalid comment
use // instead of /* */

Reply:
fixed